Deployment/src/Modules/EventHub/main.tf

```
@@ -0,0 +1,61 @@
+resource "azurerm_eventhub_namespace" "eventhub_namespace" {
+  name                = "${var.name}-namespace"
+  location            = var.location
+  resource_group_name = var.resource_group_name
+  sku                 = "Standard"
+  capacity            = 1
+  tags                = var.tags
+}
+
+resource "azurerm_eventhub" "eventhub" {
+  name                = var.name
+  namespace_name      = azurerm_eventhub_namespace.eventhub_namespace.name
+  resource_group_name = var.resource_group_name
+  partition_count     = 2
+  message_retention   = 7
+}
+
+resource "azurerm_eventhub_consumer_group" "logstash_consumer_group" {
+  name                = "logstash"
+  namespace_name      = azurerm_eventhub_namespace.eventhub_namespace.name
+  eventhub_name       = azurerm_eventhub.eventhub.name
+  resource_group_name = var.resource_group_name
+}
+
+resource "azurerm_eventhub_consumer_group" "logging_application_consumer_group" {
+  name                = "loggingApplication"
+  namespace_name      = azurerm_eventhub_namespace.eventhub_namespace.name
+  eventhub_name       = azurerm_eventhub.eventhub.name
+  resource_group_name = var.resource_group_name
+}
+
+resource "azurerm_eventhub_authorization_rule" "logstash" {
+  name                = "logstashAccessKey"
+  namespace_name      = azurerm_eventhub_namespace.eventhub_namespace.name
+  eventhub_name       = azurerm_eventhub.eventhub.name
+  resource_group_name = var.resource_group_name
+  listen              = true
+  send                = false
+  manage              = false
+}
+
+resource "azurerm_eventhub_authorization_rule" "log" {
+  name                = "logAccessKey"
+  namespace_name      = azurerm_eventhub_namespace.eventhub_namespace.name
+  eventhub_name       = azurerm_eventhub.eventhub.name
+  resource_group_name = var.resource_group_name
+  listen              = false
+  send                = true
+  manage              = false
+}
+
+resource "azurerm_storage_account" "logstashStorage" {
+  name                      = var.logstashStorageName
+  resource_group_name       = var.resource_group_name
+  location                  = var.location
+  account_kind              = "StorageV2"
+  account_tier              = "Standard"
+  account_replication_type  = "LRS"
+  access_tier               = "Hot"
+  enable_https_traffic_only = true
+}
```

This resource is missing tags

```suggestion
  tags                = var.tags
}
```